README.md

```diff
@@ -22,154 +46,149 @@ Semantic Versioning is used, and corresponds to electron versions in the followi
 - electron-pdf 1.3.x  =>  electron 1.6.x (Chromium 56, Node 7.4)
 - electron-pdf 1.2.x  =>  electron 1.4.x (Chromium 53, Node 6.5)
 
-Note: The Chromium versions employed by electron have impacts based on the functionality you may be exporting.  
-Choose the version you need based on Chromium.
+Please note that the choice of Chromium version affects the functionality you can utilize. Choose the version that aligns with your needs.
 
-Install
--------
+### Installation
 
-```
+Install Electron-PDF via npm:
+
+```bash
 npm install electron-pdf
 ```
 
-Note: If you're installing electron-pdf as root using the system level npm (vs a user-level install like with NVM) then you may need to run the following command instead:
+If you're installing as root using system-level npm (rather than a user-level install like with NVM), use the following command:
 
-```
+```bash
 sudo npm install electron-pdf -g --unsafe-perm
 ```
 
-Please see [the npm docs](https://docs.npmjs.com/misc/config#unsafe-perm) for more information.
-
-
-For gnu/linux installations without a graphical environment:
+For GNU/Linux installations without a graphical environment, you need to install xvfb and set up a virtual display:
 
 ```bash
-$ sudo apt-get install xvfb # or equivalent
-$ export DISPLAY=':99.0'
-$ Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
-$ electron-pdf ...
+sudo apt-get install xvfb # or equivalent
+export DISPLAY=':99.0'
+Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
+electron-pdf ...
 ```
 
-There is also an example docker machine [here](https://github.com/fraserxu/docker-tape-run).
+A Docker machine example is available [here](https://github.com/fraserxu/docker-tape-run).
+
+Please see [the npm docs](https://docs.npmjs.com/misc/config#unsafe-perm) for more information.
 
-Node Usage
------
-Electron PDF can be used inside of an application, or more commonly as the engine for a pdf 
-rendering service.  For instance, to handle http requests using Express.  The following snippets 
-show you how you can get started.
 
-### The application must run in an Electron process
+### Node.js Usage
+
+Electron-PDF can be integrated into your application or used as a rendering engine for a PDF service. Below are examples of usage.
+
+#### Application Setup
+
+In your `package.json`:
 
-In `package.json`
 ```json
-"start": "DEBUG=electronpdf:* electron index.js",
-"watch": "DEBUG=electronpdf:* nodemon --exec electron index.js"
+"scripts": {
+  "start": "DEBUG=electronpdf:* electron index.js",
+  "watch": "DEBUG=electronpdf:* nodemon --exec electron index.js"
+}
 ```
 
-### You can use the same instance
+#### Using Electron-PDF
 
 ```javascript
-var ElectronPDF = require('electron-pdf')
-var express = require('express')
-var bodyParser = require('body-parser')
-var app = express()
-app.use(bodyParser.json())
+const ElectronPDF = require('electron-pdf');
```

Comment:
Can we undo all these semi-colon and tabs vs spaces changes? Not interested in a code-style war 😆

Reply:
Done 2/2